core/rtw_mlme_ext.c

```diff
@@ -15168,7 +15168,9 @@ u8 set_chplan_hdl(_adapter *padapter, unsigned char *pbuf)
 	rtw_hal_set_odm_var(padapter,HAL_ODM_REGULATION,NULL,_TRUE);
 	
 #ifdef CONFIG_IOCTL_CFG80211
-	rtw_reg_notify_by_driver(padapter);
+        if ((adapter->rtw_wdev != NULL) {
```

Comment:
typo error, check the build logs
https://github.com/CGarces/rtl8189ES_linux/pull/13/checks?check_run_id=1996354548

```suggestion
        if ((padapter->rtw_wdev != NULL) {
```